### ISSUE TITLE
Sublime visible on all virtual desktops in Windows 10.

### ISSUE THREAD

**Issue by sowizz** [mikadev001/SublimeText-Overlay#17]
As in the topic. When using Sublime Overlay, a sublime window without windows decoration / overlay is visible across all virtual desktops.

As a side note, how did you accomplish those mac alike fonts in windows 10?

**Comment by mikadev001** [comment]
It is visible on VD preview, but if you switch to another desktop, the editor is not there, right?
For Mac alike font rendering I use [MacType](https://code.google.com/archive/p/mactype/downloads)

**Comment by sowizz** [comment]
No, not only on preview. When switching with key combination CTRL+WIN+LEFT/RIGHT borderless window of sublime appears on another desktops.

See:
![1](https://cloud.githubusercontent.com/assets/1541634/13600018/91fab872-e526-11e5-8726-2cdb142dcd0e.PNG)

![2](https://cloud.githubusercontent.com/assets/1541634/13600025/9632b91c-e526-11e5-9094-f512fe5d610a.PNG)
